wowup-electron/src/assets/i18n/de.json

```diff
@@ -114,22 +114,22 @@
       "POSITIVE_BUTTON": "Ja"
     },
     "INSTALL_FROM_URL": {
-      "ADDON_URL_INPUT_LABEL": "Addon URL",
-      "ADDON_URL_INPUT_PLACEHOLDER": "z.B. GitHub- oder WowInterface URL",
+      "ADDON_URL_INPUT_LABEL": "Addon-URL",
+      "ADDON_URL_INPUT_PLACEHOLDER": "z.B. GitHub- oder WoWInterface-URL",
       "CLOSE_BUTTON": "Schließen",
-      "DESCRIPTION": "Wenn Du ein Addon direkt per URL installieren möchtest, füge diese unten in der Zeile ein.",
+      "DESCRIPTION": "Wenn Du ein Addon direkt von einer URL installieren möchtest, füge sie unten in die Zeile ein.",
       "DOWNLOAD_COUNT": "{textCount} {count, plural, one{Download} other{Downloads}} auf {provider}",
       "ERROR": {
         "FAILED_TO_CONNECT": "Keine Verbindung mit der API möglich, warte ein wenig und versuche es erneut.",
-        "INSTALL_FAILED": "Beim Versuch das Addon zu installieren ist etwas schief gelaufen, bitte versuche es erneut.\n\nWenn diese Nachricht weiterhin erscheint, kannst Du auf unserem Discord im Kanal #wowup-support nach Hilfe fragen.",
+        "INSTALL_FAILED": "Beim Versuch das Addon zu installieren ist etwas schief gelaufen, bitte versuche es erneut.\n\nWenn diese Nachricht weiterhin erscheint, kannst Du auf unserem Discord im Channel #wowup-support nach Hilfe fragen.",
         "INVALID_URL": "Die eingetragene URL ist ungültig. Beispiele für gültige URLs sind:\n\t- https://github.com/teelolws/Altoholic-Retail\n\t- https://www.wowinterface.com/downloads/info25610-8.3-014.html\n\t- https://www.curseforge.com/wow/addons/altoholic",
-        "NO_ADDON_FOUND": "Es wurde kein Addon gefunden. Stelle sicher, dass die URL zur richtigen Website führt.\n\nSolltest Du versuchen ein Addon von GitHub zu installieren, stelle sicher, dass das Repository ein Release Etikett mit einem ZIP Archiv hat, welches das Addon enthält.",
-        "TITLE": "Addon Installation fehlgeschlagen"
+        "NO_ADDON_FOUND": "Es wurde kein Addon gefunden. Stelle sicher, dass die URL zur richtigen Website führt.\n\nSolltest Du versuchen ein Addon von GitHub zu installieren, stelle sicher, dass das Repository Release-Tags mit einem ZIP-Archiv, welches das Addon enthält, besitzt.",
```

Comment:
@Bruncky ist schon gemerged - aber kann es sein, dass in dem Satz hier etwas nicht stimmt?

Reply:
Welchen Teil meinst du? Vom Inhalt her passt das soweit ich weiß, der Satzbau könnte aber verbessert werden!